addons/zeus/functions/fnc_ui_removeObjects.sqf

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author: Fisher
+ * Removes all objects in given radius for all curators.
+ *
+ * Arguments:
+ * 0: Dummy controls group <CONTROL>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+ 
+#include "script_component.hpp"
+ 
+disableSerialization;
```

Comment:
If they don't run in scheduled environment, then `disableSerialization` should be removed as unscheduled environment does not support serialization to begin with.